doc/dev/developer_guide/essentials.rst

```diff
@@ -7,9 +7,9 @@ to know.
 Leads
 -----
 
-The Ceph project is led by Sage Weil. In addition, each major project
-component has its own lead. The following table shows all the leads and
-their nicks on `GitHub`_:
+The Ceph project was created by Sage Weil and is led by the Ceph Leadership
+Team (CLT). In addition, each major project component has its own lead. The
+following table shows all the leads and their nicks on `GitHub`_:
```

Comment:
s/shows all /lists/

Reply:
As I read over this, I have some ideas about making the English better in general:

> The Ceph project was created by Sage Weil and is led by the Ceph Leadership Team. Each major component of Ceph has its own lead. This table lists the leads and their nicks on GitHub:

Reply:
Possibly even s/has its own lead/has a lead/

Reply:
Fair enough, `own` doesn't bother me too much, but I see your motivation.